app/templates/marketing/terms_of_service.html

```diff
@@ -88,6 +86,9 @@ <h2 class="ui header" id="our_standards">Our Standards</h2>
             <li>
                 Sharing links, copies, or instructions how to obtain any copyrighted content in a way that infringes on the copyright holder's rights (as provided in the copyright owners's country).
             </li>
+            <li>
+                Discussing divisive topics that are not immediately relevant to the Python community in channels not dedicated toward such divisive topics (politics, political figures, public figures, tangents that lead toward related discourse, etc.).
```

Comment:
I would also add `Religion` here.

Reply:
Agree with Religion.

I'd argue "tangents that lead..." is too generic and gives admins too much licence.